src/database/mod.rs

```diff
@@ -134,6 +134,9 @@ pub trait Database: BatchOperations {
     ///
     /// It should insert and return `0` if not present in the database
     fn increment_last_index(&mut self, keychain: KeychainKind) -> Result<u32, Error>;
+
+    /// Force changes to be written to disk, returning the number of bytes written
```

Comment:
Needs to be updated to match last change:
```suggestion
    /// Force changes to be written to disk
```